helpers/lbryApi.js

```diff
@@ -22,7 +24,7 @@ module.exports = {
     logger.debug(`lbryApi >> Publishing claim to "${publishParams.name}"`);
     return new Promise((resolve, reject) => {
       axios
-        .post('http://localhost:5279/lbryapi', {
+        .post('http://' + apiUri + ':' + apiPort + '/lbryapi', {
```

Comment:
Thanks for the PR!  

Define the url string once at the top so the string doesn't have to be constructed repeatedly.  Also, the `/lbryapi` can be dropped.  
E.g. 
`const lbryUrl = 'http://' + apiUri + ':' + apiPort;`
...
`axios.post(lbryUrl, ... `

If you can make this change I'll merge :)

Reply:
Also, let me know your discord handle so I can send you an lbc tip